src/Evolution/Systems/Cce/Equations.hpp

```diff
@@ -0,0 +1,1020 @@
+      Spectral::Swsh::Tags::Derivative<Tags::Beta,
+                                       Spectral::Swsh::Tags::EthEth>,
+      Spectral::Swsh::Tags::Derivative<Tags::Beta,
+                                       Spectral::Swsh::Tags::EthEthbar>,
+      Spectral::Swsh::Tags::Derivative<
+          Spectral::Swsh::Tags::Derivative<Tags::J,
+                                           Spectral::Swsh::Tags::Ethbar>,
+          Spectral::Swsh::Tags::Eth>,
+      Spectral::Swsh::Tags::Derivative<Tags::Multiplies<Tags::J, Tags::Jbar>,
+                                       Spectral::Swsh::Tags::EthEthbar>,
+      Spectral::Swsh::Tags::Derivative<Tags::Multiplies<Tags::J, Tags::Jbar>,
+                                       Spectral::Swsh::Tags::Eth>,
+      Spectral::Swsh::Tags::Derivative<Tags::Q, Spectral::Swsh::Tags::Eth>,
+      Spectral::Swsh::Tags::Derivative<Tags::U, Spectral::Swsh::Tags::Eth>,
+      Spectral::Swsh::Tags::Derivative<
+          Tags::Multiplies<Tags::Ubar, Tags::Dy<Tags::J>>,
+          Spectral::Swsh::Tags::Eth>,
+      Spectral::Swsh::Tags::Derivative<Tags::Dy<Tags::J>,
+                                       Spectral::Swsh::Tags::Ethbar>,
+      Spectral::Swsh::Tags::Derivative<Tags::J,
+                                       Spectral::Swsh::Tags::EthbarEthbar>,
+      Spectral::Swsh::Tags::Derivative<Tags::J, Spectral::Swsh::Tags::Ethbar>,
+      Spectral::Swsh::Tags::Derivative<
+          Tags::Multiplies<Tags::Jbar, Tags::Dy<Tags::J>>,
+          Spectral::Swsh::Tags::Ethbar>,
+      Spectral::Swsh::Tags::Derivative<Tags::JbarQMinus2EthBeta,
+                                       Spectral::Swsh::Tags::Ethbar>,
+      Spectral::Swsh::Tags::Derivative<Tags::Q, Spectral::Swsh::Tags::Ethbar>,
+      Spectral::Swsh::Tags::Derivative<Tags::U, Spectral::Swsh::Tags::Ethbar>>;
+};
+template <>
+struct tags_needed_for_integrand_computation_impl<
+    Tags::RegularIntegrand<Tags::H>,
+    TagsCategory::IndependentOfBondiIntegration> {
+  using type = tmpl::list<Tags::DuRDividedByR, Tags::EthRDividedByR, Tags::K,
+                          Tags::OneMinusY, Tags::R>;
+};
+
+// - Tags Linear operator that acts on H
+template <>
+struct tags_needed_for_integrand_computation_impl<
+    Tags::LinearFactor<Tags::H>, TagsCategory::TagsToSwshDifferentiate> {
+  using type = tmpl::list<Tags::Dy<Tags::J>, Tags::J>;
+};
+template <>
+struct tags_needed_for_integrand_computation_impl<
+    Tags::LinearFactor<Tags::H>, TagsCategory::IndependentOfBondiIntegration> {
+  using type = tmpl::list<Tags::OneMinusY>;
+};
+template <>
+struct tags_needed_for_integrand_computation_impl<Tags::LinearFactor<Tags::H>,
+                                                  TagsCategory::Temporary> {
+  using type = tmpl::list<
+      ::Tags::SpinWeighted<::Tags::TempScalar<0, ComplexDataVector>, 2>>;
+};
+
+// - Tags for Linear operator that acts on the conjugate of H
+template <>
+struct tags_needed_for_integrand_computation_impl<
```

Comment:
These type aliases, for each term, should be a single struct, and the collections of tags should probably be placed in a separate file.